packages/main/src/Slider.hbs

```diff
@@ -0,0 +1,36 @@
+<div 
```

Comment:
Overall: class convention is OpenUI5, use the dashes ui5-something (rather than sapMSomething)